File: app/src/main/java/io/github/ovso/righttoknow/data/network/model/video/Snippet.java
package io.github.ovso.righttoknow.data.network.model.video;

import java.util.Date;
import lombok.Getter;

@Getter public class Snippet {
  private Date publishedAt;
  private String channelId;
  private String title;
  private String description;
  private SearchThumbnails thumbnails;
  private String channelTitle;
  private String liveBroadcastContent;
}

File: app/src/main/java/io/github/ovso/righttoknow/data/network/model/video/Snippet.kt
package io.github.ovso.righttoknow.data.network.model.video

import java.util.*

class Snippet(
  val publishedAt: Date,
  val channelId: String,
  val title: String,
  val description: String,
  val thumbnails: SearchThumbnails,
  val channelTitle: String,
  val liveBroadcastContent: String
)